tools/windows-node-installer/README.md

```diff
@@ -88,6 +91,11 @@ The IDs of created instance and security group are saved to the `window-node-ins
 ### Destroying Windows instances:
 
 ```bash
+wni aws destroy --help
+Destroy all resources specified in 'windows-node-installer.json' file in the current or specified directory, 
```

Comment:
`Destroy all security groups and instances specified in 'windows-node-installer.json' file in the current or specified directory`
You are not destroying any other resources.

Reply:
Uhmm not really. For different vendors, we might delete more than just the security groups and instances. For example Azure, from our last discussion, deletes network card and etc. Since those belong to a single instance and can be found by instance id, the information is not stored in the json file but the resource is still deleted. This information is discussed last time and I am not aware of any changes. @vinaykns Therefore, I am trying to be not very specific here.

Reply:
You writing docs for aws. Why are you bringing Azure into the picture.

Reply:
This is the document for `wni` which I believe should be generic enough to accommodate all the platforms that we support. I understand right now it might not be my responsibility to take care of the Azure platform, but since it is something that we have discussed already, I assume we want to reflect them here.

Reply:
You still don't get my point. Saying `wni aws destroy --help` and then talking about Azure doesn't make sense. If you want this to be generic, then you have to craft this differently.